src/usr/local/www/system_hasync.php

```diff
@@ -173,6 +174,15 @@ interfaces_sync_setup();
 ))->setHelp('Enter the webConfigurator password of the system entered above for synchronizing the configuration.%1$s' .
 			'Do not use the Synchronize Config to IP and password option on backup cluster members!', '<br />');
 
+$section->addInput(new Form_Checkbox(
+	'adminsync',
+	'Synchronize admin',
+	'synchronize admin accounts and autoupdate sync password.',
+	($pconfig['adminsync'] === 'on'),
+	'on'
+))->setHelp('By default, admin accounts does not sync, and you can have different passwords on both nodes.%1$s' .
```

Comment:
Reword to fix grammar and avoid addressing the user directly ("you"). For example:

    'By default, the admin account does not synchronize, and each node may have a different admin password.%1$s' .
    'This option automatically updates XMLRPC Remote System Password when the password is changed on the Remote System Username account.'

Reply:
ok, fixed